scene/gui/option_button.cpp

```diff
@@ -69,6 +69,9 @@ void OptionButton::_notification(int p_what) {
 					case DRAW_HOVER:
 						clr = get_theme_color(SNAME("font_hover_color"));
 						break;
+					case DRAW_HOVER_PRESSED:
+						clr = get_theme_color(SNAME("font_hover_pressed_color"));
```

Comment:
Should add explicitly to the default theme then, as it is only defined for the base class, but `OptionButton` type redefines all the other colors.